lib/simplifyExpression/multiplyFractionsSearch/index.js

```diff
@@ -22,10 +22,12 @@ function multiplyFractions(node) {
   if (!Node.Type.isOperator(node) || node.op !== '*') {
     return Node.Status.noChange(node);
   }
+
+  // we need to use the verbose syntax for `some` here because isFraction
+  // can take more than one parameter
```

Comment:
omg I love that you added this comment - I was just thinking about how someone might come back to this later and change it, so good call